scripts/jest-package.js

```diff
@@ -7,18 +9,20 @@ import runInPackage from './utils/base-package';
  * @returns {Promise}
  */
 export default function jestPackage(pkgName, pkgPath) {
-  return runInPackage({
```

Comment:
@bzang this was failing and no matter what params I passed got it to work. This way works now, not sure if it just never got updated with other utils changes.